components/utils/ShapeList.js

```diff
@@ -533,7 +519,7 @@ const ShapeList = ({
                       <ShapeActions className="shape-actions">
                         <ShapeActionsPrimary>
                           <span
-                            onClick={(event, shapeId) =>
+                            onClick={(event) =>
```

Comment:
Why this change is required in this PR?

Reply:
it is not required. I just saw some unused variables and wanted to get rid of it